fdbserver/workloads/BlobGranuleCorrectnessWorkload.actor.cpp

```diff
@@ -432,23 +452,37 @@ struct BlobGranuleCorrectnessWorkload : TestWorkload {
 			           blob.first.size());
 		}
 
+		// Create key sorted index.
+		std::vector<Key> blobKeys;
+		std::unordered_map<Key, int> keyChunkIndex;
+		for (int i = 0; i < blob.second.size(); i++) {
+			Key beginKey = blob.second[i].keyRange.begin;
+			blobKeys.push_back(beginKey);
+		}
+		std::sort(blobKeys.begin(), blobKeys.end(), [](const auto& lhs, const auto& rhs) { return lhs.compare(rhs); });
```

Comment:
Oddly, I had correctness failures where it wasn't returned in sorted order.

Reply:
That itself is I think a bug then? The chunks for a read, if it returns multiple, should always be disjoint, consecutive, and in order.
The read validation would fail if this wasn't the case iirc, but maybe it's not checking that somewhere